examples/cookbook/lists/long_lists/lib/main.dart

```diff
@@ -28,6 +28,13 @@ class MyApp extends StatelessWidget {
         // #docregion ListView
         body: ListView.builder(
           itemCount: items.length,
+          //when the listview is scrolled using the scrollbar all the way to
+          //the bottom of the view,
+          //chrome gets unresponsive
+          // specifying prototypeItem fixes the issue because the scrolling
+          //machinery can make use of the foreknowledge of the children's
+          //extent
+          prototypeItem: const ListTile(title: Text('prototype item')),
```

Comment:
The snippets update tool, mentioned in the README needs to be run to update the page. I'd also argue that instead of using a comment for the exposition, that it should actually be as text in https://github.com/flutter/website/tree/main/src/cookbook/lists/long-lists.md

Reply:
I believe that's done.